.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2.1
```

Comment:
I tested this CircleCI config after enabling it on my account (see https://app.circleci.com/pipelines/github/blimmer/ts-app-env?branch=chore%2Ffreshen-repo&filter=all). I think you'll need to flip the switch on your repo once merged from the fork.